nautobot/dcim/filters/__init__.py

```diff
@@ -688,9 +694,6 @@ def _power_outlets(self, queryset, name, value):
     def _interfaces(self, queryset, name, value):
         return queryset.exclude(interface_templates__isnull=value)
 
-    def _pass_through_ports(self, queryset, name, value):
-        return queryset.exclude(front_port_templates__isnull=value, rear_port_templates__isnull=value)
-
```

Comment:
`FrontPortTemplate.rear_port_template` is not nullable so if you have a `FrontPortTemplate`, you have pass through ports. I omitted this filter entirely on `Module` and `ModuleType` for this reason.

Reply:
Should the change above be using `rear_port_templates` instead? Since a device can have rear-ports without any front-ports, but not the reverse?